gusto/timeloop.py

```diff
@@ -152,6 +157,11 @@ def run(self, t, tmax, pickup=False):
 
 class AdvectionTimestepper(BaseTimestepper):
 
+    def __init__(self, state, advection_dict, physics_list=[]):
+
+        super(AdvectionTimestepper, self).__init__(state, advection_dict)
+        self.physics_list = physics_list
```

Comment:
I did think about doing this. I didn't know if we wanted the physics to permanently be part of the AdvectionTimestepper... @colinjcotter mentioned yesterday that we might want to reorganise the timeloop, so this might change soon anyway?